### PR TITLE
import mesos_cli as a subpackage in paasta_tools

### PR DESCRIPTION
**Pull request by Rob-Johnson**
rather than forking, let's bring this into the paasta
project. This brings in everything that was needed for
the itests to pass - we'll save clearing out the bits we don't
need for the future.

Note that I've had to change [this](https://github.com/Yelp/paasta/compare/vendor-mesos-cli?expand=1#diff-b32f029cd959a6adcdcdb50bbdf326d2R43). I think I need to update the license at the top of the file to [make a note of what's changed?](https://tldrlegal.com/license/apache-license-2.0-(apache-2.0))

itests pass locally for me. I'll iterate from here to try and pick out the bits that we actually need (but follow up with separate PRs)

### CODE REVIEW

**Comment by Rob-Johnson** [comment]
the itests are failing because of all the lambdas. I guess I'll fix them

**Comment by Rob-Johnson** [comment]
please pay attention to the [second commit](https://github.com/Yelp/paasta/pull/748/commits/64e4b21d24379b56af82127db41d49732c707991) when reviewing - I'd like reassurance that the functionality is the same.

File: paasta_tools/mesos/zookeeper.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+# Helper for testing
+client_class = kazoo.client.KazooClient
+
+
+@contextlib.contextmanager
+def client(*args, **kwargs):
+    zk = client_class(*args, **kwargs)
+    try:
+        zk.start(timeout=TIMEOUT)
+    except kazoo.handlers.threading.KazooTimeoutError:
```
Comment (Rob-Johnson) [inline paasta_tools/mesos/zookeeper.py]: Note that this is the fix for https://github.com/Yelp/paasta/issues/353

**Comment by solarkennedy** [comment]
Ship! This isn't so bad.

**Pull request merged**